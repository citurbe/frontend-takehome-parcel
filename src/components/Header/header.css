.header {
  background-color: #e9573f;
  color: white;
  height: 60px;
  margin-bottom: 20px;
  padding-top: 30px;
  padding-left: 20px;
}

h4 {
  margin-block-start: 0;
}
