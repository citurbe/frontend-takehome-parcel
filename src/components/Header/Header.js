import React from 'react';
import './header.css';

const Header = () => {
  return (
    <div className="header">
      <h4>Ruby Gems Search</h4>
    </div>
  );
};

export default Header;
