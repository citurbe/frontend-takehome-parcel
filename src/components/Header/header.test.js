import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';

test('header renders correctly', () => {
  const component = renderer.create(<Header />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
