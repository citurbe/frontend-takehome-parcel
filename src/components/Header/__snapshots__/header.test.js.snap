// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders correctly 1`] = `
<div
  className="header"
>
  <h4>
    Ruby Gems Search
  </h4>
</div>
`;
